runelite-client/src/main/java/net/runelite/client/plugins/inventoryviewer/InventoryViewerOverlay.java

```diff
@@ -105,4 +146,9 @@ private BufferedImage getImage(Item item)
 		ItemComposition itemComposition = itemManager.getItemComposition(item.getId());
 		return itemManager.getImage(item.getId(), item.getQuantity(), itemComposition.isStackable());
 	}
+
+	private BufferedImage getImage(int id, int quantity)
```

Comment:
what is the point of this method.

Reply:
This used to have more in it. Thanks for spotting it